app/assets/stylesheets/active_admin/_typography.scss

```diff
@@ -98,3 +85,8 @@ th,td,caption { padding: 4px 10px 4px 5px; }
 .last       { margin-right:0; padding-right:0; }
 .top        { margin-top:0; padding-top:0; }
 .bottom     { margin-bottom:0; padding-bottom:0; }
+
+hr {
+  border: none;
+  border-bottom: 1px solid #E8E8E8;
```

Comment:
Color literals like `#E8E8E8` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#E8E8E8` should be written as `#e8e8e8`